style-guide.md

```diff
@@ -0,0 +1,56 @@
+# WDL Style Guide
```

Comment:
Perhaps we don't do it in this version, but I think some details on what a "good" documentation header block looks like would be helpful. Most of the time, people just want a template they can fill in.